mscore/scoreview.cpp

```diff
@@ -4129,6 +4130,8 @@ void ScoreView::cmdAddText(Tid tid)
       if (s) {
             _score->select(s, SelectType::SINGLE, 0);
             _score->endCmd();
+            if (customTid != Tid::DEFAULT && customTid != tid)
+                  s->initTid(customTid);
```

Comment:
Wouldn't it be better to move this code before the `endCmd()` call? Then `Score::update()` will be called with a correct `Tid` for the element already.

Reply:
I think you're right.  Will update.